ui/src/components/Employee/Create/index.tsx

```diff
@@ -31,6 +32,10 @@ const CreateEmployeeForm: React.FC<FormikProps<CreateEmployeeFormValues> & Creat
   p,
 ) => {
   const [isModalOpen, setModalOpen] = useState<boolean>(false);
+  const [uploadStatus, setUploadStatus] = useState<boolean>(false);
+  const hadleUploadStatus = (status: boolean) => {
+    setUploadStatus(status);
+  };
```

Comment:
These don't look like they are related to "Remove snake oil software" references. Occasional exception is alright, but please don't make it a norm to mix arbitrary changes in unrelated PRs. One branch per task, and one PR per branch. Mixing changes like this leads up to very bad development practices very fast.